css/index.css

```diff
@@ -1,3 +1,159 @@
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
 
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+body {
+	width: 900px;
+    margin: auto;
+    /* border: 1px dashed red; */
+}
+header {
+	display: flex;
+	/* width: 100%; */
+	justify-content: space-between;
+    margin: 20px 0 20px 0;
+    /* border: 1px dashed blue; */
+}
+nav a {
+    /* border: 1px dashed gold; */
+	text-decoration: none;
+	color: #BBB;
+	margin: 5px 15px 5px 15px;
+}
+.top, .mid-top, .mid-bottom {
+	display: flex;
+	justify-content: space-around;
+    margin: 35px 0;
+    /* border: 1px dashed green;     */
+}
+.top {
+	align-items: center;
+}
+.top-left {
+	text-align: center;
+}
+h1 {
+	font-size: 350%;
+}
+#get-started {
+    border: 1px solid black;
+    border-radius: 5%;
+    padding: 5px 10px;
+}
+#get-started:hover {
+    background-color: #DDD;
+}
+.middle-img {
+	width: 100%;
+}
+.mid-bottom div, .mid-top div {
+    margin: 10px;
+}
+.contact {
+	margin: 35px 0 35px 0;
+}
+footer {
+	text-align: center;
+	margin: 35px;
+}
+
+@media (max-width: 800px) {
+    header {
+        flex-direction: column-reverse;
+        align-items: center;
+    }
+    header nav a {
+        margin: 25px;
+    }
+    header img {
+        max-width: 100%;
+        height: auto; 
+        display: block;
+        margin-bottom: 25px;
+    }
+    .top img {
+        display: none;
+    }
+}
+@media (max-width: 500px) {
+    header {
+        flex-flow: column-reverse;
+        align-items: center;
+    }
+    header nav {
+        display: flex;
+        flex-flow: column;
```

Comment:
```suggestion
        flex-flow: column;
        width: 100%;
```

Reply:
fills the screen space without forcing a horizontal scroll with padding property (removed below)